From 43912a370760233ecb8aecc78587e1a122b6edf5 Mon Sep 17 00:00:00 2001
From: Shivani Raval <sraval@habana.ai>
Date: Tue, 10 Sep 2024 13:43:35 +0300
Subject: [PATCH] [FSW-6362] scripts: modified from mgmt to mgmt_io

Renamed mgmt to mgmt_io in post image scripts and
its file parameter.

Change-Id: I7c486a1f1091ff5af5125e6870864f6f283e0610
Signed-off-by: Shivani Raval <sraval@habana.ai>
---

diff --git a/board/habanalabs/fs1_arc/fs1-post-image-fit-arc.sh b/board/habanalabs/fs1_arc/fs1-post-image-fit-arc.sh
index a73d0aa..5e8aa04 100755
--- a/board/habanalabs/fs1_arc/fs1-post-image-fit-arc.sh
+++ b/board/habanalabs/fs1_arc/fs1-post-image-fit-arc.sh
@@ -43,10 +43,10 @@
 ITS_SED_PARAMS+="s/@ARC_PID_DCCM_LO@/${ARC_PID_DCCM_LO}/g;"
 ITS_SED_PARAMS+="s/@ARC_PID_ICCM_HI@/${ARC_PID_ICCM_HI}/g;"
 ITS_SED_PARAMS+="s/@ARC_PID_ICCM_LO@/${ARC_PID_ICCM_LO}/g;"
-ITS_SED_PARAMS+="s/@ARC_MGMT_DCCM_HI@/${ARC_MGMT_DCCM_HI}/g;"
-ITS_SED_PARAMS+="s/@ARC_MGMT_DCCM_LO@/${ARC_MGMT_DCCM_LO}/g;"
-ITS_SED_PARAMS+="s/@ARC_MGMT_ICCM_HI@/${ARC_MGMT_ICCM_HI}/g;"
-ITS_SED_PARAMS+="s/@ARC_MGMT_ICCM_LO@/${ARC_MGMT_ICCM_LO}/g;"
+ITS_SED_PARAMS+="s/@ARC_MGMT_IO_DCCM_HI@/${ARC_MGMT_IO_DCCM_HI}/g;"
+ITS_SED_PARAMS+="s/@ARC_MGMT_IO_DCCM_LO@/${ARC_MGMT_IO_DCCM_LO}/g;"
+ITS_SED_PARAMS+="s/@ARC_MGMT_IO_ICCM_HI@/${ARC_MGMT_IO_ICCM_HI}/g;"
+ITS_SED_PARAMS+="s/@ARC_MGMT_IO_ICCM_LO@/${ARC_MGMT_IO_ICCM_LO}/g;"
 
 # Replace ARC core IDs
 ITS_SED_PARAMS+="s/@FIT_CORE_ID_IO0_PID@/${FIT_CORE_ID_IO0_PID}/g;"
diff --git a/board/habanalabs/fs1_arc/fs1-zephyr.its b/board/habanalabs/fs1_arc/fs1-zephyr.its
index 779218b..5dac561 100644
--- a/board/habanalabs/fs1_arc/fs1-zephyr.its
+++ b/board/habanalabs/fs1_arc/fs1-zephyr.its
@@ -39,28 +39,28 @@
 		};
 		*/
 
-		zephyr-mgmt-iccm {
-			description = "Zephyr mgmt ICCM";
-			data = /incbin/("@BINS_DIR_PATH@@DEV_NAME@-zephyr-mgmt-iccm.bin.le");
+		zephyr-mgmt_io-iccm {
+			description = "Zephyr mgmt io die ICCM";
+			data = /incbin/("@BINS_DIR_PATH@@DEV_NAME@-zephyr-mgmt_io-iccm.bin.le");
 			type = "standalone";
 			arch = "arc";
 			core-id = <@FIT_CORE_ID_IO0_MGMT@>;
 			compression = "none";
-			loadaddr = <@ARC_MGMT_ICCM_HI@ @ARC_MGMT_ICCM_LO@>;
-			startaddr = <@MGMT_STARTADDR_HI@ @MGMT_STARTADDR_LO@>;
+			loadaddr = <@ARC_MGMT_IO_ICCM_HI@ @ARC_MGMT_IO_ICCM_LO@>;
+			startaddr = <@MGMT_IO_STARTADDR_HI@ @MGMT_IO_STARTADDR_LO@>;
 			hash {
 				algo = "@SHA_VERSION@";
 			};
 		};
 
-		zephyr-mgmt-dccm {
-			description = "Zephyr mgmt DCCM";
-			data = /incbin/("@BINS_DIR_PATH@@DEV_NAME@-zephyr-mgmt-dccm.bin.le");
+		zephyr-mgmt_io-dccm {
+			description = "Zephyr mgmt io die DCCM";
+			data = /incbin/("@BINS_DIR_PATH@@DEV_NAME@-zephyr-mgmt_io-dccm.bin.le");
 			type = "standalone";
 			arch = "arc";
 			core-id = <@FIT_CORE_ID_IO0_MGMT@>;
 			compression = "none";
-			loadaddr = <@ARC_MGMT_DCCM_HI@ @ARC_MGMT_DCCM_LO@>;
+			loadaddr = <@ARC_MGMT_IO_DCCM_HI@ @ARC_MGMT_IO_DCCM_LO@>;
 			hash {
 				algo = "@SHA_VERSION@";
 			};
@@ -73,8 +73,8 @@
 		preboot-conf {
 			load-type = <0>; /* FIT_LOAD_TYPE_RUN */
 			description = "Habanalabs device preboot";
-			loadables = /* "zephyr-pid-iccm", "zephyr-pid-dccm", */ "zephyr-mgmt-iccm",
-				    "zephyr-mgmt-dccm";
+			loadables = /* "zephyr-pid-iccm", "zephyr-pid-dccm", */ "zephyr-mgmt_io-iccm",
+				    "zephyr-mgmt_io-dccm";
 		};
 	};
 };
diff --git a/board/habanalabs/fs1_arc/post-image-fit.sh b/board/habanalabs/fs1_arc/post-image-fit.sh
index 34524d4..36cc7c3 100755
--- a/board/habanalabs/fs1_arc/post-image-fit.sh
+++ b/board/habanalabs/fs1_arc/post-image-fit.sh
@@ -22,8 +22,8 @@
 	# TEMPORARY: arcpid isn't ready yet, so comment-out the following lines. Otherwise, the script fails.
 	#cp ${BR2_EXTERNAL}/output/${DEV_NAME}_arc/images/${DEV_PREFIX}-zephyr-pid-iccm.bin ${FILEPATH}/${DEV_PREFIX}-zephyr-pid-iccm.bin.le
 	#cp ${BR2_EXTERNAL}/output/${DEV_NAME}_arc/images/${DEV_PREFIX}-zephyr-pid-dccm.bin ${FILEPATH}/${DEV_PREFIX}-zephyr-pid-dccm.bin.le
-	cp ${BR2_EXTERNAL}/output/${DEV_NAME}_arc/images/${DEV_PREFIX}-zephyr-mgmt-iccm.bin ${FILEPATH}/${DEV_PREFIX}-zephyr-mgmt-iccm.bin.le
-	cp ${BR2_EXTERNAL}/output/${DEV_NAME}_arc/images/${DEV_PREFIX}-zephyr-mgmt-dccm.bin ${FILEPATH}/${DEV_PREFIX}-zephyr-mgmt-dccm.bin.le
+	cp ${BR2_EXTERNAL}/output/${DEV_NAME}_arc/images/${DEV_PREFIX}-zephyr-mgmt_io-iccm.bin ${FILEPATH}/${DEV_PREFIX}-zephyr-mgmt_io-iccm.bin.le
+	cp ${BR2_EXTERNAL}/output/${DEV_NAME}_arc/images/${DEV_PREFIX}-zephyr-mgmt_io-dccm.bin ${FILEPATH}/${DEV_PREFIX}-zephyr-mgmt_io-dccm.bin.le
 
 	return
 }
@@ -33,13 +33,13 @@
 	hl_extract_zephyr_elf_sections
 popd
 
-MGMT_STAT=${STATS_DIR}/zephyr_mgmt.stat
+MGMT_IO_STAT=${STATS_DIR}/zephyr_mgmt_io.stat
 # TEMPORARY: arcpid isn't ready yet, so comment-out the following line. Otherwise, the script fails.
 #PID_STAT=${STATS_DIR}/zephyr_pid.stat
 
 # Extract the start address from stat file
-ENTRY_ADDRESS=$(cat $MGMT_STAT | grep "Entry point address" | head -1)
-MGMT_STARTADDR=$(echo $ENTRY_ADDRESS | awk -F: '{ print $2 }' | xargs)
+IO_ENTRY_ADDRESS=$(cat $MGMT_IO_STAT | grep "Entry point address" | head -1)
+MGMT_IO_STARTADDR=$(echo $IO_ENTRY_ADDRESS | awk -F: '{ print $2 }' | xargs)
 # TEMPORARY: arcpid isn't ready yet, so comment-out the following line. Otherwise, the script fails.
 #ENTRY_ADDRESS=$(cat $PID_STAT | grep "Entry point address" | head -1)
 PID_STARTADDR=$(echo $ENTRY_ADDRESS | awk -F: '{ print $2 }' | xargs)
@@ -48,9 +48,9 @@
 # ARC_PID_ICCM_LO to get 32 bit address and XX_STARTADDR_HI
 # is same as ARC_XX_ICCM_HI.
 PID_STARTADDR_LO=$((ARC_PID_ICCM_LO | PID_STARTADDR))
-MGMT_STARTADDR_LO=$((ARC_MGMT_ICCM_LO | MGMT_STARTADDR))
+MGMT_IO_STARTADDR_LO=$((ARC_MGMT_IO_ICCM_LO | MGMT_IO_STARTADDR))
 
 ITS_SED_PARAMS+="s/@PID_STARTADDR_HI@/${ARC_PID_ICCM_HI}/g;"
 ITS_SED_PARAMS+="s/@PID_STARTADDR_LO@/${PID_STARTADDR_LO}/g;"
-ITS_SED_PARAMS+="s/@MGMT_STARTADDR_HI@/${ARC_MGMT_ICCM_HI}/g;"
-ITS_SED_PARAMS+="s/@MGMT_STARTADDR_LO@/${MGMT_STARTADDR_LO}/g;"
+ITS_SED_PARAMS+="s/@MGMT_IO_STARTADDR_HI@/${ARC_MGMT_IO_ICCM_HI}/g;"
+ITS_SED_PARAMS+="s/@MGMT_IO_STARTADDR_LO@/${MGMT_IO_STARTADDR_LO}/g;"
